### PR TITLE
fix: adjust deviation check configs in manifests

### PR DESCRIPTION
**Pull request by cehali**
It turned out during my fight with dry-runs that deviation config for a specific token is not taken under consideration, in check it always returns `undefined` for custom deviation config so it uses default one

### CODE REVIEW

File: packages/oracle-node/src/manifest/validate-manifest.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ const CustomUrlsSchema = z.object({
 
 const TokenSchema = z.object({
   source: z.array(z.string()).optional(),
-  deviationCheck: z.string().optional(),
+  deviationCheck: DeviationCheckSchema.optional(),
```
Comment (koonopek) [inline packages/oracle-node/src/manifest/validate-manifest.ts:36]: Why this change?
Also why we don't change in node code

**Review by koonopek — APPROVED** [review]

**Review by rpiszczatowski — APPROVED** [review]
